mlkem/api.h

```diff
@@ -0,0 +1,66 @@
+#ifndef API_H
```

Comment:
Please add the SPDX identifiers already, please? 
```
// SPDX-License-Identifier: Apache-2.0
```

Reply:
I plan to separate the SPDX identifiers in a separate commit. 

Btw, I believe for MLKEM it is: 
```
// SPDX-License-Identifier: Apache-2.0 or CC0-1.0
```

right?

Reply:
No. We need to state in the text the original license which indeed is Apache-2.0 or CC0-1.0.
But we do need to state here that we are using it under Apache-2.0 and all modifications are going to be covered by that. I learned that only this week, so I'll still have to make the changes accordingly.

Reply:
Nice. I'll add the Apache 2.0 License ASAP. CC0-1.0 is problematic to some organizations.